src/cosmospy/_transaction.py

```diff
@@ -21,87 +26,149 @@ class Transaction:
     """
 
     def __init__(
-        self,
-        *,
-        privkey: bytes,
-        account_num: int,
-        sequence: int,
-        fee: int,
-        gas: int,
-        fee_denom: str = "uatom",
-        memo: str = "",
-        chain_id: str = "cosmoshub-4",
-        hrp: str = DEFAULT_BECH32_HRP,
-        sync_mode: SyncMode = "sync",
+            self,
+            *,
+            privkey: bytes,
+            account_num: int,
+            sequence: int,
+            fee: int,
+            gas: int,
+            fee_denom: str = "uatom",
+            fee_account: str = "",
```

Comment:
`fee_account` wasn't needed before. Do we need it now? Is it new functionality that can be added in another PR?

Reply:
It was indeed a new functionality implementing the feegrant module. I implemented this because I needed it for a project and it is just one attribute that has to be passed to the transaction. We can make a separate PR for this with a better feegrant implementation then.